code/src/main/kotlin/com/expediagroup/sdk/lodgingconnectivity/configuration/LodgingConnectivityLogMasking.kt

```diff
@@ -0,0 +1,27 @@
+package com.expediagroup.sdk.lodgingconnectivity.configuration
+
+import com.expediagroup.sdk.core.logging.masking.LogMaskingFeature
+
+/**
+ * Object implementing the MaskingConfiguration interface.
+ * Initializes and adds masking patterns based on global and path-specific fields at startup time automatically.
+ */
+internal val LodgingConnectivityLogMasking =
```

Comment:
Wonder if we should follow the same pattern with adding `Abstract*` prefix for classes that are expected to be implemented in the product SDKs.
We can possibly rename the base class to `AbstractLogMaskingFeature` and the implementation in the product SDK would be just `LogMaskingFeature` instead of `LodgingConnectivityLogMasking`.